### PR TITLE
Adding security and network doc for the agent

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Add a link to the list of DD IP range for agent data collection and integrations

### Preview link
https://docs-staging.datadoghq.com/gus/ip-ranges/agent/network/

### Note

This KB is now outdated: https://help.datadoghq.com/hc/en-us/articles/209618853-Can-I-whitelist-the-IP-addresses-for-data-coming-from-Datadog-via-Webhook-and-integrations-

And this one too: https://github.com/DataDog/dd-agent/wiki/Network-Traffic-and-Proxy-Configuration

We should remove its content and point to this new FAQ instead.

### CODE REVIEW

File: content/agent/faq/datadog-ip-ranges.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Datadog IP ranges
+kind: faq
+---
+
+All data sent to Datadog or coming from Datadog Webhooks and integrations, such as those with ticketing systems like ServiceNow and JIRA, are coming from a set of static IPs:
```
Comment (AndrewMcBurney) [inline content/agent/faq/datadog-ip-ranges.md]: Should `Webhooks` be capitalized?

Comment (AndrewMcBurney) [reply content/agent/faq/datadog-ip-ranges.md]: Should it be this instead?

> All data sent to Datadog or coming from Datadog webhooks and integrations, such as those with ticketing systems like ServiceNow and JIRA, are coming from a set of static IPs:

Comment (jyee) [reply content/agent/faq/datadog-ip-ranges.md]: This statement is confusing and wrong (as it's currently worded). `All data sent to Datadog... are coming from a set of static IPs`. That makes it sound like the Datadog Agent is sending *from* those IPs, when it's actually sending *to* those IPs.

Do we really need this placeholder page or can we just link the Agent FAQ directly to the wiki page (https://github.com/DataDog/dd-agent/wiki/Network-Traffic-and-Proxy-Configuration)? If we link directly we don't have to duplicate content and it saves the user one click to get to the pertinent info.

File: content/agent/network.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+  * **`8125/udp`**: dogstatsd
+
+* **For Agent version < 6.0**: 
+
+  * **`17123/tcp`**: agent forwarder, used to buffer traffic in case of network
+  splits between the agent and Datadog
+  * **`17124/tcp`**: optional graphite adapter
+
+    Unless `non_local_traffic` is set to true. These ports are available on localhost:
```
Comment (masci) [inline content/agent/network.md]: Should this be un-indented and part of the main paragraph?

Comment (l0k0ms) [reply content/agent/network.md]: @masci  the `non_local_traffic` variable applies to agent 6 too ?

Comment (masci) [reply content/agent/network.md]: ah my bad, in 6.0 the first two cases don't apply as said but for dogstasd the corresponding flag was renamed to `dogstatsd_non_local_traffic`

File: content/agent/network.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+}
+```
+
+## Open Ports
+
+### Agent v6
+
+  * **`8125/udp`**: dogstatsd
+
+  Unless `dogstatsd_non_local_traffic` is set to true. These ports are available on localhost:
```
Comment (masci) [inline content/agent/network.md]: nit: it's only one port, "This port is available on localhost:"

Comment (l0k0ms) [reply content/agent/network.md]: good catch

**Pull request merged**